advisor-go/src/quixo/server/server.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+	"net/http"
+	"strconv"
+)
+
+const port int = 8001
```

Comment:
`port` could be declared as an environment variable (to be injected via the `docker-composer.yml`)